### PR TITLE
Added Azure DevOps NodeGoat Prioritize yml example

### PR DESCRIPTION
**Pull request by lukebroganws**

### CODE REVIEW

File: Prioritize/JavaScript/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- script: |
+    npm install
+  displayName: 'npm install'
+  
+- script: |
+    curl -LJO https://github.com/whitesource/unified-agent-distribution/raw/master/standAlone/wss_agent.sh; 
+    echo Unified Agent downloaded successfully;
+    chmod +x wss_agent.sh;
+    ./wss_agent.sh -appPath ./package.json -d ./ -product ADO-NodeGoat -project ADO-NodeGoat;
```
[the remark this answers is not in the document]
Comment (samq-ws) [reply Prioritize/JavaScript/azure-pipelines.yml:26]: We are avoiding the CLI args in these examples as it confuses end users when not all commands are available via CLI args

File: Prioritize/JavaScript/azure-pipelines.yml [hunk 2]
```diff
@@ -25,11 +25,12 @@ steps:
     chmod +x wss_agent.sh;
```
Comment (samq-ws) [inline Prioritize/JavaScript/azure-pipelines.yml:25]: Should be using the following commands and not the wss_agent.sh 
```
    curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
    echo Unified Agent downloaded successfully
    java -jar wss-unified-agent.jar -appPath ./package.json -d ./
```

File: Prioritize/JavaScript/azure-pipelines.yml [hunk 3]
```diff
@@ -25,11 +25,12 @@ steps:
...
     ./wss_agent.sh -appPath ./package.json -d ./ -product ADO-NodeGoat -project ADO-NodeGoat;
   env:
-    WS_APIKEY: <key>
+    WS_APIKEY: $(APIKEY)
+    WS_USERKEY: $(USERKEY)
     WS_WSS_URL: https://saas-eu.whitesourcesoftware.com/agent
```
Comment (samq-ws) [inline Prioritize/JavaScript/azure-pipelines.yml:30]:
```
WS_WSS_URL: $(WSS_URL)
```
This should not be hardcoded as the URL changes

File: Prioritize/JavaScript/azure-pipelines.yml [hunk 4]
```diff
@@ -25,11 +25,12 @@ steps:
...
     WS_ENABLEIMPACTANALYSIS: true
     WS_RESOLVEALLDEPENDENCIES: false
-    REQUIREKNOWNSHA1: false
+    WS_REQUIREKNOWNSHA1: false
     WS_FILESYSTEMSCAN: false
     WS_NPM_RESOLVEDEPENDENCIES: true
     WS_NPM_RESOLVELOCKFILE: false
```
Comment (samq-ws) [inline Prioritize/JavaScript/azure-pipelines.yml:36]: ```WS_NPM_RUNPRESTEP: false ```` is unnecessary as this is the default https://whitesource.atlassian.net/wiki/spaces/WD/pages/1544880156/Unified+Agent+Configuration+Parameters#JavaScript

File: Prioritize/JavaScript/azure-pipelines.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukebroganws) [reply Prioritize/JavaScript/azure-pipelines.yml:26]: yep, that makes sense :) thanks!

**Review by tabac-ws — DISMISSED** [review]

**Review by tabac-ws — APPROVED** [review]

**Review by samq-ws — APPROVED** [review]

**Pull request merged**